src/layout.v3.tsx

```diff
@@ -121,84 +121,81 @@ export const Footer: React.StatelessComponent<{}> = ({}) => {
     <footer className='footer white ph3 ph4-ns pt4 pt5-ns pb3 bg-sky'>
       <section className='flex-ns items-start-ns justify-around-ns pr3-m pr7-l'>
         <div className='dn db-l'>
-          <a href='/' className='logo w4 db'>
-            <img src='assets/logo.svg' alt='Learnk8s' />
+          <a href='/' className='logo w4 db' title='Learnk8s logo'>
+            <Learnk8sLogo />
           </a>
-          <p>Learn Kubernetes the smart way</p>
+          <p>The Kubernetes training company</p>
         </div>
         <div className='dn db-ns'>
           <h2 className='ttu f6'>Services</h2>
           <ul className='list pl0'>
-            <li className='mv2'>
-              <a href='/training' className='link white'>
-                Training
-              </a>
-            </li>
-            <li className='mv2'>
-              <a href='/academy' className='link white'>
-                Academy
-              </a>
-            </li>
-            <li className='mv2'>
-              <a href='/consulting' className='link white'>
-                Consulting
-              </a>
-            </li>
+            {[
+              ['Training', '/training'],
+              ['Online courses', '/Academy'],
```

Comment:
I think it is lowercase `/academy`, but it works.

Reply:
Good spot